tensorflow_datasets/image/celebahq.py

```diff
@@ -68,20 +69,23 @@ def __init__(self, resolution, **kwargs):
         version=v2,
         **kwargs)
     self.resolution = resolution
-    self.file_name = "data%dx%d.tar" % (resolution, resolution)
+    self.file_name = "data%dx%d" % (resolution, resolution)
+    self.extensions = resource._KNOWN_EXTENSIONS
 
 
 class CelebAHq(tfds.core.GeneratorBasedBuilder):
   """Celeba_HQ Dataset."""
 
   MANUAL_DOWNLOAD_INSTRUCTIONS = """\
-  manual_dir should contain multiple tar files with images (data2x2.tar,
-  data4x4.tar .. data1024x1024.tar).
+  manual_dir should contain the following compressed files with images.
+  (data2x2.zip, data4x4.tar .. data1024x1024.bz2).
+  You can find some of the celebaHQ data here:
+  https://drive.google.com/drive/folders/11Vz0fqHS2rXDb5pprgTjpD7S2BAJhi1P
```

Comment:
Where is this url coming from. Is it an official url ?

Reply:
I found it on fast.ai forum.
https://forums.fast.ai/t/download-celeba-hq-dataset/45873/3

Reply:
Does that mean this is not the original files ? We should always download from the original source. Mirrors from external users can be corrupted so we should not trust them.

Could you update the dataset to download the official data ?

Reply:
If anything we should contact the official team so they upload a better version of their files.

Reply:
Okay, sure

Reply:
@Conchylicultor I could do that but the code says
```
# The current celebahq generation code depends on a concrete version of 
# pillow library and cannot be easily ported into tfds.
```

Reply:
Then the users should process the data themselves externally from the celeb_a official website and then put this data into the MANUAL_DIR